From 4363eb5c5aec80cfc2b5c36d8555d38697a14041 Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Tue, 4 Apr 2017 11:26:37 +0300
Subject: [PATCH 0989/1345] fix: dts: a8040: enable AXI monitors

commit  59ab514d0b4aee40ca2df5fe610a901e6dd15dc8 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

AXI monitors were disabled in device tree, as part of A8040-DB
board bring up, due to some issue with the monitors probing.
AXI monitors now function properly, so enable all monitors
for A8040.

Change-Id: If579d420a469e3df6bff79139cce313998605e3e
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/38461
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040.dtsi |   48 --------------------------
 1 file changed, 48 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040.dtsi b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
index 04604c6..94a9589 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
@@ -101,18 +101,6 @@
 			etr@812000 {
 				status = "disabled";
 			};
-			axim-ddr-rd@840000 {
-				status = "disabled";
-			};
-			axim-ddr-wr@841000 {
-				status = "disabled";
-			};
-			axim-sb-rd@848000 {
-				status = "disabled";
-			};
-			axim-sb-wr@849000 {
-				status = "disabled";
-			};
 			smmu: iommu@5000000 {
 				mmu-masters = <&usb3h0 0x483>,
 					      <&usb3h1 0x484>;
@@ -136,24 +124,6 @@
 	cp110-master {
 		config-space@f2000000 {
 			/* Temp: disable all Coresight nodes */
-			axim-cp-rd@3c5000 {
-				status = "disabled";
-			};
-			axim-cp-wr@3c6000 {
-				status = "disabled";
-			};
-			axim-ppv2-rd@3c0000 {
-				status = "disabled";
-			};
-			axim-ppv2-wr@3c1000 {
-				status = "disabled";
-			};
-			axim-hb1-rd@3c8000 {
-				status = "disabled";
-			};
-			axim-hb1-wr@3c9000 {
-				status = "disabled";
-			};
 			ppv22@000000 {
 				dma-coherent;
 			};
@@ -205,24 +175,6 @@
 	cp110-slave {
 		config-space@f4000000 {
 			/* Temp: disable all Coresight nodes */
-			axim-cp-rd@3c5000 {
-				status = "disabled";
-			};
-			axim-cp-wr@3c6000 {
-				status = "disabled";
-			};
-			axim-ppv2-rd@3c0000 {
-				status = "disabled";
-			};
-			axim-ppv2-wr@3c1000 {
-				status = "disabled";
-			};
-			axim-hb1-rd@3c8000 {
-				status = "disabled";
-			};
-			axim-hb1-wr@3c9000 {
-				status = "disabled";
-			};
 			ppv22@000000 {
 				dma-coherent;
 			};
-- 
1.7.9.5

